Paramore.Brighter.Locking.Azure/AzureBlobLockingProvider.cs

```diff
@@ -11,9 +11,7 @@ public class AzureBlobLockingProvider(AzureBlobLockingProviderOptions options) :
     private readonly BlobContainerClient _containerClient = new BlobContainerClient(options.BlobContainerUri, options.TokenCredential);
     private readonly ILogger _logger = ApplicationLogging.CreateLogger<AzureBlobLockingProviderOptions>();
 
-    private readonly Dictionary<string, string> _leases = new Dictionary<string, string>();
-
-    public async Task<bool> ObtainLockAsync(string resource, CancellationToken cancellationToken)
+    public async Task<string?> ObtainLockAsync(string resource, CancellationToken cancellationToken)
```

Comment:
Our style is to document public methods, using XML comments, as this shows up in Intellisense

Reply:
Thank you, removed

Reply:
Comments included